js/app/src/api_docs/ApiDocs.svelte

```diff
@@ -182,11 +187,11 @@
 				<InstallSnippet {current_language} />
 
 				{#if Object.keys(info.named_endpoints).length}
-					<h2 class="header">Named Endpoints</h2>
+					<h2 class="header">Endpoints</h2>
```

Comment:
The deployed space says "undefined" right underneath the Enpoints header

<img width="180" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/ae946660-fc7b-41a6-a99e-27354d3466b0">

Reply:
I actually don't see the footer at all in the deployed Space? How are you seeing this?

<img width="1317" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/b04c6c84-957f-4065-af6a-095566b9397e">

Reply:
If you meant the test Space, I just installed the latest version of this branch and its working for me: 
<img width="1050" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/b0687b52-67d0-462e-920b-cc093279bcb3">